app/models/partner.rb

```diff
@@ -26,6 +26,8 @@ class Partner < ApplicationRecord
       .order(:name)
   }
 
+  scope :unapproved, -> { where(status: "Pending").or(where(status: "Awaiting Review")) }
```

Comment:
Nice scope -- we should probably only be looking for `Awaiting Review` as Pending is the default status and when a new diaper bank uploads their list of partners they are going to have a lot of notifications that aren't going to be super helpful for them.